input.php

```diff
@@ -0,0 +1,16 @@
+<html>
+    <head></head>
+    <body>
+        <form action='hidden.php' method='POST'>
+            <input type='text' name='name' />
+            <label for="name">What's their name?</label><p>
```

Comment:
whats the random opening `<p>` tag? this is not valid HTML

also the `for` attribute on the label tag actually relates to the `id` attribute on an input, not the `name` attribute. so you'll need to add IDs to the inputs